packages/drivers/local-driver/src/localDocumentStorageService.ts

```diff
@@ -131,25 +133,31 @@ export class LocalDocumentStorageService implements IDocumentStorageService {
 
 	/**
 	 * Strips the tree or any subtree of data if it has a groupId.
+	 * Collect the blobIds to keep in the snapshot.
 	 *
 	 * @param tree - The tree to strip of loading groupIds
 	 * @returns a tree that has trees with groupIds that are empty
 	 */
 	private async stripTreeOfMissingLoadingGroupIds(
```

Comment:
This is more now of an update all relevant blobs function. Totally not important. This function doesn't really strip anything, it just gets. 

Here's a mind teaser, if a tree looks like "" -> "g1" -> "g2", and our request gets us "g2" technically, we would get an empty tree. This is complete food for thought here. It isn't technically a requirement, but it's a pretty nasty edge case.

Reply:
Renamed the function.